contracts/swaps/connectors/SwapsImplSovrynSwap.sol

```diff
@@ -107,11 +117,6 @@ contract SwapsImplSovrynSwap is State, ISwapsImpl {
                 "insufficient source tokens provided."
             );
             minReturn = requiredDestTokenAmount;
-        } else if (sourceTokenAmountUsed > 0) {
-            /// For some reason the Sovryn swap network tends to return a bit less than the expected rate.
-            minReturn = sovrynSwapNetwork.rateByPath(path, sourceTokenAmountUsed).mul(995).div(
-                1000
-            );
```

Comment:
is this not an issue anymore?

Reply:
Yes, querying minReturn to the AMM at the same block is kind of useless, so we chnage it to 1 instead as the minReturn